src/nginx/varify/varify.go

```diff
@@ -15,6 +18,11 @@ func main() {
 	filename := os.Args[1]
 	localModulePath := os.Args[2]
 	globalModulePath := os.Args[3]
+	resolvConfPath := "/etc/resolv.conf"
```

Comment:
Can we declare this and defaultNameserver in a top level var block, so its easy to pick them out?

Reply:
Ok, array of strings?
What should happen when getting `nameservers` throws an exception? 
Warning + empty array?